vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/ReferenceableIdentifiersImportIT.java

```diff
@@ -401,6 +422,138 @@ public void testImportDupPolicyLegacy() throws RepositoryException, IOException,
         assertNull(duplicateNode);
     }
 
+    @Test
+    public void testInstallPackage_CREATE_NEW_ID() throws Exception {
+        test(IdConflictPolicy.CREATE_NEW_ID, null, null, true, true);
+    }
+
+    @Test
+    public void testInstallPackage_FAIL() throws Exception {
+        test(IdConflictPolicy.FAIL, RepositoryException.class, null, false, false);
+    }
+
+    @Test
+    public void testInstallPackage_FORCE_REMOVE_CONFLICTING_ID() throws Exception {
+        test(IdConflictPolicy.FORCE_REMOVE_CONFLICTING_ID, null, null, false, false);
+    }
+
+    @Test
+    public void testInstallPackage_LEGACY() throws Exception {
+        test(IdConflictPolicy.LEGACY, RepositoryException.class, IllegalStateException.class, false, false);
+    }
+
+    private void test(IdConflictPolicy policy, Class<?> expectedException, Class<?> expectedRootCause, boolean expectNewId,
```

Comment:
can we rename to `assertSomething` and replace the `Something` by the thing which is actually asserted here? Only actual test methods being executed directly by JUnit should have `test` in the name.

Reply:
Yep.